lib/config_helper.rb

```diff
@@ -7,6 +7,7 @@ module ConfigHelper
 
   def setup_action_mailer(config)
     config.action_mailer.preview_path = Rails.root.join('spec', 'mailers', 'previews')
+    config.action_mailer.show_previews = Rails.env.development? || FeatureFlipper.staging_email?
```

Comment:
this should make both http://localhost:3000/rails/mailers and https://staging-api.va.gov/rails/mailers/ available to use.  This should not enable this in production

Reply:
I would propose this and related changes for the mailer preview functionality in staging be broken out into a separate PR please

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/5557